.github/workflows/dependencies/gcc.sh

```diff
@@ -23,3 +23,4 @@ sudo apt-get install -y \
 
 python3 -m pip install -U pip setuptools wheel
```

Comment:
Re: https://github.com/ECP-WarpX/WarpX/pull/3106
```suggestion
python3 -m pip install -U pip setuptools wheel
python3 -m pip install -U cmake
```